src/script/zcash_script.h

```diff
@@ -99,6 +119,32 @@ EXPORT_SYMBOL int zcash_script_verify(
     uint32_t consensusBranchId,
     zcash_script_error* err);
 
+/// Returns 1 if the input nIn of the serialized transaction pointed to by
+/// txTo correctly spends the scriptPubKey pointed to by scriptPubKey under
+/// the additional constraints specified by flags. Must be used for V5 transactions;
+/// may also be used for previous versions.
+///
+/// allPrevOutputs must point to the concatenation of all outputs from previous
+/// transactions that are spent by the inputs of the given transaction.
+/// The outputs must be encoded as specified by Bitcoin (value followed by
+/// pk_script, including a leading CompactSize).
```

Comment:
Make the same changes here as are made to `zcash_script_new_precomputed_tx_v5()` to address review comments.

Reply:
Done with b459f197f